slackapi/slackbot-bolt.mjs

```diff
@@ -0,0 +1,312 @@
+            "element": {
+              "type": "plain_text_input",
+              "action_id": "input_name",
+              "placeholder": {
+                "type": "plain_text",
+                "text": "Book name"
+              }
+            },
+            "label": {
+              "type": "plain_text",
+              "text": "Title"
+            }
+          },
+          {
+            "type": "input",
+            "block_id": "input_book_author",
+            "element": {
+              "type": "plain_text_input",
+              "action_id": "input_author",
+              "placeholder": {
+                "type": "plain_text",
+                "text": "Author name"
+              }
+            },
+            "label": {
+              "type": "plain_text",
+              "text": "Author",
+              "emoji": true
+            }
+          },
+          {
+            "type": "input",
+            "block_id": "input_book_abstract",
+            "element": {
+              "type": "plain_text_input",
+              "action_id": "input_abstract",
+              "multiline": true
+            },
+            "label": {
+              "type": "plain_text",
+              "text": "Abstract"
+            },
+            "hint": {
+              "type": "plain_text",
+              "text": "Description of the book"
+            }
+          },
+          {
+            "type": "input",
+            "block_id": "input_book_publication",
+            "element": {
+              "type": "datepicker",
+              "initial_date": "1990-04-28",
+              "placeholder": {
+                "type": "plain_text",
+                "text": "Select the publication date",
+                "emoji": true
+              },
+              "action_id": "input_publication"
+            },
+            "label": {
+              "type": "plain_text",
+              "text": "Publication date",
+              "emoji": true
+            }
+          },
+          {
+            "type": "input",
+            "block_id": "input_book_cover",
+            "element": {
+              "type": "plain_text_input",
+              "action_id": "input_cover",
+              "placeholder": {
+                "type": "plain_text",
+                "text": "https://url.image"
+              }
+            },
+            "label": {
+              "type": "plain_text",
+              "text": "Cover URL",
+              "emoji": true
+            },
+            "optional": true
+          }
+        ],
+        "type": "modal",
+        "callback_id": 'view_book'
+      }
+    });
+  }
+  catch (error) {
+    console.error(error);
+  }
+});
+
+slackApp.view('view_book', async ({ ack, body, view, client }) => {
+  await ack();
+
+  const newBook = view['state']['values'];
+  const user = body['user']['id'];
+  const userName = body['user']['username'];
+  let msg = '';
+
+  const userInfo = await slackClient.users.info({
+    user: user
```

Comment:
I think that in this case it is not necessary to write user:user, with only user works
```suggestion
    user,
```